src/BlockchainWriter/Router.ts

```diff
@@ -50,11 +51,15 @@ export class Router {
     }
   }
 
-  blockDownloadedConsumer = async (blockDownloaded: BlockDownloaded): Promise<void> => {
+  blockDownloadedConsumer = async (blockDownloaded: BlockDownloaded): Promise<void> =>
     await this.claimController.setBlockInformationForTransactions(
       getTxnIds(blockDownloaded.poetBlockAnchors),
       blockDownloaded.block,
     )
+
+  onPurgeStaleTransactions = async (): Promise<void> => {
+    await this.claimController.purgeStaleTransactions()
+    await this.messaging.publish(this.exchange.anchorNextHashRequest, '')
```

Comment:
We don't want to publish this message. The service already does this on an interval.

Reply:
ack